packages/template-retail-react-app/app/components/item-variant/item-attributes.jsx

```diff
@@ -68,6 +112,67 @@ const ItemAttributes = ({includeQuantity, currency, ...props}) => {
                 </Text>
             )}
 
+            {!productBundleIsLoading && productBundleData && !productBundleVariantData && (
+                <Box>
+                    {productBundleData?.bundledProducts.map(({product, quantity}) => (
+                        <Box marginTop={2} key={product.id}>
+                            <Text fontSize="sm" color="gray.700" as="b">
+                                {product?.name}
+                            </Text>
+                            <Text fontSize="sm" color="gray.700">
+                                {intl.formatMessage({
+                                    defaultMessage: 'Qty',
+                                    id: 'add_to_cart_modal.label.quantity'
+                                })}
+                                : {quantity}
+                            </Text>
+                        </Box>
+                    ))}
+                </Box>
+            )}
+
+            {!bundleVariantIsLoading && productBundleVariantData && (
+                <Box>
+                    <Text fontSize={15} marginTop={3} fontWeight={500}>
+                        {intl.formatMessage({
+                            defaultMessage: 'Selected Options',
+                            id: 'item_attributes.label.selected_options'
+                        })}
+                        :
+                    </Text>
+                    {productBundleVariantData?.map(
+                        ({variationValues, name: productName, quantity, id}) => {
+                            return (
+                                <Box key={id} marginTop={2}>
+                                    <Text fontSize="sm" color="gray.700" as="b">
+                                        {productName}
+                                    </Text>
+                                    <Text fontSize="sm" color="gray.700">
+                                        {intl.formatMessage({
+                                            defaultMessage: 'Qty',
+                                            id: 'add_to_cart_modal.label.quantity'
+                                        })}
+                                        : {quantity}
+                                    </Text>
+                                    {Object.keys(variationValues).map((key) => {
+                                        const selectedVariant = `${key}: ${variationValues[key]}`
+                                        return (
+                                            <Text
+                                                fontSize="sm"
+                                                color="gray.700"
+                                                key={selectedVariant}
+                                            >
+                                                {selectedVariant}
```

Comment:
Should this be localized text?

Reply:
great call out, didn't think about that but checked on `RefArchGlobal` and it seems that the API returns the variant information that we display in the language of the selected locale, so we don't have to pull this out for localization